portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,135 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US" dir="ltr">
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="wisth=device-width, initial-scale=1">
+
+    <title>Arrington's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <header>
+          <h1>arrington.</h1>
+          <!-- maybe add a leaderboard ad - they're 728 x 90 -->
+          <nav>
+              <a>home</a>
+              <a>about</a>
+              <a>projects</a>
+              <a>contact</a>
+              <a>skills</a>
+              </ul>
+          </nav>
+      </header>
+    
+
+    <!-- elements to add:
+        contacting arrington
+        arrington's interests (done)
+        arrington's details/skills (done)
+        arrington's schools (done)
+        arrington's companies (done)
+        arrington's blurbs
+        arrington's friend space
+        favorite song
+        projects + relevant coursework
+        my header or whatever it's called (done)
+         -->
```

Comment:
I think the convention for when you plan to add something in the future is to prefix it with `TODO:` like `<!-- TODO: add details about school -->`. At google we sometimes also even include our username in the TODO so other people can know who is going to look at it like `<!-- TODO(aokeke): add details about school -->`, In this case I don't think you need your username since it is just you, though, but having the `TODO`, lets you easily Ctrl+F for all places where work needs to be done and some IDEs even have special highlighting for comments prefixed by `TODO`

Reply:
Gotcha, I just added it!